src/components/util/text-field-with-adornment.js

```diff
@@ -79,13 +80,14 @@ const TextFieldWithAdornment = (props) => {
         getClearAdornment,
         isFocused,
         getTextAdornment,
-        classes,
+        classes.inputRight,
     ]);
 
     const withStartAdornmentText = useCallback(() => {
-        return value
+        return value !== '' || isFocused
             ? {
-                  startAdornment: isFocused && getTextAdornment('start'),
+                  startAdornment:
+                      value && isFocused && getTextAdornment('start'),
```

Comment:
Should be only `getTextAdornment('start')` here, no need to check `value && isFocused`.
Should be `endAdornment: value && clearable && getClearAdornment('end')` on the following line

Reply:
done